packages/joy-api/src/client.ts

```diff
@@ -72,19 +74,19 @@ async function parseVideoInputs(video: Video): Promise<[VideoInputMetadata, Vide
     size: videoSize,
   }
 
-  const thumbnailPhotoStream = await getThumbnailAsset(video.thumbnails.default)
+  const thumbnailPhotoStream = await getThumbnailAsset(video.thumbnails)
   const { hash: thumbnailPhotoHash, size: thumbnailPhotoSize } = await computeFileHashAndSize(thumbnailPhotoStream)
-
   const thumbnailPhotoMetadata: DataObjectMetadata = {
     ipfsHash: thumbnailPhotoHash,
     size: thumbnailPhotoSize,
   }
+
   assets.media = videoMetadata
   assets.thumbnailPhoto = thumbnailPhotoMetadata
   return [videoInputMetadata, assets]
 }
 
-async function getThumbnailAsset(thumbnailUrl: string) {
-  const response = await axios.get<Readable>(thumbnailUrl, { responseType: 'stream' })
+async function getThumbnailAsset(thumbnails: Thumbnails) {
+  const response = await axios.get<Readable>(thumbnails.medium, { responseType: 'stream' })
```

Comment:
Could you add a small comment that the medium thumbnail is the only one that has a 16/9 aspect ratio? Someone could change that to high, standard, or default for some reason and could be not aware, that the format o those is different.

Reply:
Done in [6471486](https://github.com/Joystream/youtube-synch/pull/83/commits/647148661dd5699e6fbc0b22dfc57376b1a7b723)